docs/src/custom-relu.md

```diff
@@ -0,0 +1,131 @@
+```julia
+using Statistics
+using DiffOpt
+using Flux
+using Flux: onehotbatch, onecold, crossentropy, throttle
+using Base.Iterators: repeated
+using OSQP
+using JuMP
+using ChainRulesCore
+```
+
+
+```julia
+## prepare data
+imgs = Flux.Data.MNIST.images()
+labels = Flux.Data.MNIST.labels();
+
+# Preprocessing
+X = hcat(float.(reshape.(imgs, :))...) #stack all the images
+Y = onehotbatch(labels, 0:9); # just a common way to encode categorical variables
+
+test_X = hcat(float.(reshape.(Flux.Data.MNIST.images(:test), :))...)
+test_Y = onehotbatch(Flux.Data.MNIST.labels(:test), 0:9)
+
+# float64 to float16, to save memory
+X = convert(Array{Float16,2}, X) 
+test_X = convert(Array{Float16,2}, test_X)
+
+X = X[:, 1:1000]
+Y = Y[:, 1:1000];
+```
+
+
+
+## Custom Relu
+
+
+```julia
+"""
+    relu method for a Matrix
+"""
+function myRelu(y::AbstractMatrix{T}; model = Model(() -> diff_optimizer(OSQP.Optimizer))) where {T}
+    x̂ = zero(y)
```

Comment:
verify that the fancy superscripts like `x̂` display correctly in the documentation

Reply:
changed